codalab/rest/bundles.py

```diff
@@ -426,6 +425,67 @@ def _fetch_locations():
     return dict(data=uuids_to_locations)
 
 
+@get('/bundle_stores/', apply=AuthenticatedProtectedPlugin())
+def _fetch_bundle_stores():
+    """
+    Fetch the bundle stores available to the user.
+
+    Returns a dictionary in which the keys are the bundle store uuids, and the values
```

Comment:
Is this the standard format we use in other REST API endpoints to return multiple items?

Reply:
I mean, does it have to be the same format? I think a mapping of uuid->object is pretty intuitive/good

Reply:
Yes, we want consistency in our REST API endpoints.